src/index.scss

```diff
@@ -107,3 +107,44 @@ $fa-font-path: "~font-awesome/fonts";
   margin-top: 17px;
   margin-left: 17px;
 }
+
+.course-tabs-navigation {
+  border-bottom: solid 1px #eaeaea;
+
+  .nav a,
+  .nav button {
+    &:hover {
+      background-color: $light-400;
+    }
+  }
+
+  .nav a {
+    &:not(.active):hover {
+      background-color: $light-400;
+      border-bottom: none;
+    }
+  }
+}
+
+.nav-underline-tabs {
+  margin: 0 0 -1px;
+
+  .nav-link {
+    border-bottom: 4px solid transparent;
+    border-top: 4px solid transparent;
+    color: $gray-700;
+
+    // temporary until we can remove .btn class from dropdowns
+    border-left: 0;
+    border-right: 0;
```

Comment:
move all Scss related to the navigation bar into a separate independent Scss file.